CRM/Event/Form/Registration/Confirm.php

```diff
@@ -45,6 +45,29 @@ class CRM_Event_Form_Registration_Confirm extends CRM_Event_Form_Registration {
    */
   protected $submittableMoneyFields = ['total_amount', 'net_amount', 'non_deductible_amount', 'fee_amount', 'tax_amount', 'amount'];
 
+  /**
+   * @var int|mixed
```

Comment:
@eileenmcnaughton We may _think_ this should be an `int` (or a `float`?), but the fact is that for the most part it's not. So i think this should be typed as `array`, or simply `mixed` alone.

Reply:
I'll just not type it I think